src/app/body/create-new-team/create-new-team.component.ts

```diff
@@ -97,6 +103,45 @@ export class CreateNewTeamComponent implements OnInit {
     });
   }
 
+  changeLabels(labelName){
+    console.log(labelName);
+    if(labelName == "Marketing"){
+      console.log("check")
+      this.type = marketingLabelsTempelate.type;
+      this.statusLabels = marketingLabelsTempelate.statusLabels;
+      this.difficultyLabels = marketingLabelsTempelate.difficultyLabels;
+      this.priorityLabels = marketingLabelsTempelate.priorityLabels;
+      this.milestoneStatusLabels = marketingLabelsTempelate.milestoneStatusLabels
+    }
+    else if(labelName == "Development"){
+      console.log("check1");
+      this.type = developmentLabelsTempelate.type;
+      this.statusLabels = developmentLabelsTempelate.statusLabels;
+      this.difficultyLabels = developmentLabelsTempelate.difficultyLabels;
+      this.priorityLabels = developmentLabelsTempelate.priorityLabels;
+      this.milestoneStatusLabels = developmentLabelsTempelate.milestoneStatusLabels;
```

Comment:
can be simplified with a variable.
Introduce variable like the selected label template, and just change the selected label template in this.